leo_bringup/scripts/leo_system

```diff
@@ -16,13 +15,21 @@ def shutdownCallback(x):
     rospy.loginfo("Shutdown command invoked")
     os.system("shutdown -h now")
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
+    namespace = os.getenv('ROS_NAMESPACE')
+    msg = "" if namespace == None else str(namespace)
```

Comment:
You should not rely on the `ROS_NAMESPACE` environment variable. Instead use [rospy.get_namespace()](http://docs.ros.org/en/melodic/api/rospy/html/rospy.names-module.html#get_namespace) method to get the namespace the node was launched in.